test/e2e/run.sh

```diff
@@ -1199,6 +1203,12 @@ host-get-vm-config "$vm" || host-set-vm-config "$vm" "$distro" "$cri"
 
 if [ -z "$VM_IP" ] || [ -z "$VM_SSH_USER" ]; then
     screen-create-vm
+else
```

Comment:
I'd like to avoid running bootstrap commands on every launch --- and would actually try to avoid running it automatically when using an existing (not created by run.sh) host unless explicitly requested.

Maybe we could add `vm-reinit()` that would run bootstrap commands and then setup proxies? (First bootstrap then proxies would be the order in which these commands are executed for new vm's created by run.sh, so we might be a bit more safe if we'd execute them in this order in reinit, too.)

An existing way to execute arbitrary test script (including calling `vm-setup-proxies` and/or `vm-reinit`) are the hooks. Like:
```
$ on_vm_online='vm-bootstrap; vm-setup-proxies' ./run.sh interactive
```

But I'm fine having a new variable in `run.sh help` to assist new user initializing an existing host.

Reply:
Sure, I am fine with that. The command line gets quite complex with different options and combinations, just wondering if user is able to memorize all options that would need to be set.